### ISSUE TITLE
Support Frigate Docking

### ISSUE THREAD

**Issue by cloaked2222** [HW-PlayersPatch/Development#230]
In the past fighters docking with support frigates was too slow. Has anything been done about this? I don't recall.

**Comment by Novaras** [comment]
No, it still sucks massive balls and makes repair vettes really annoying to use.
With that said, there are two main problems:
1. strike can't get into dock position with a moving supporter, and will just trail it at the supporter's move speed until it slows down or they die (or the dock order is cancelled)
2. shooting fighters off the sides of a supporter is amazingly easy, they're basically emp'd while docked in terms of how easy it is to shoot them.
